lib/argon2.js

```diff
@@ -17,21 +17,15 @@
         Argon2i: 1
     };
 
-    var scriptLoadedPromise;
-
+    var loadArgon2WrapperPromise;
     function loadScript(src) {
         return new Promise(function (resolve, reject) {
-            if (typeof importScripts === 'function') {
-                importScripts(src);
```

Comment:
Why removing support of `importScripts` ? By removing this you prevent the use of this library in a web worker